src/components/common/Layout/GuestLayout/index.js

```diff
@@ -1,14 +1,31 @@
-import React, { memo } from 'react';
+import React, { memo, useState } from 'react';
 import { node } from 'prop-types';
 
+import ContactForm from 'components/common/ContactForm';
 import Menu from '../../HamburgerMenu';
 import SocialMediaContainer from '../../SocialMediaContainer';
+import { createQuestion } from 'state/actions/contactActions';
+import { useDispatch } from 'hooks';
 
 const GuestLayout = ({ children }) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const createQuestionRequest = useDispatch(createQuestion);
+
+  const toggleModalOpen = () => {
+    setIsModalOpen(!isModalOpen);
```

Comment:
there is a hook called `useToggle` in `hooks` folder that you can use for this

Reply:
mmm I haven't found it on the hooks folder. Maybe it's from another template/project? 🤔

Reply:
You are right, I have created it in my project 😄 my bad! 

Here is what I did, in case you want to take a look:

```
import { useCallback, useState } from "react";

const useToggle = (initialState = false) => {
  const [state, setState] = useState(initialState);

  const toggle = useCallback(() => setState((state) => !state), []);

  return [state, toggle];
};

export default useToggle;
```

Reply:
Haha np. Great, thank you, already added it to this project!